src/braket/aws/aws_device.py

```diff
@@ -203,7 +203,7 @@ def run(
             **aws_quantum_task_kwargs,
         )
 
-    def run_batch(
+    def run_batch(  # noqa: D417
```

Comment:
why do we need this noqa? (undocumented param)

Reply:
`*aws_quantum_task_args` these will be fixed later. In a lot of instances, I am thinking of the best way to define `*` args.

Reply:
Got it. Glad fixing them is part of the plan, as I think it's often quite confusing what they are for without documentation

Reply:
Yes, this is something that as I went through the code I really had to jog my memory for what can go and what was acceptable values. If a new person comes across these, this can be a pretty jarring dive into the code